### PR TITLE
A4A > Marketplace: Implement animation for the illustration background when switching tabs

### PR DESCRIPTION
**Pull request by yashwin**
Closes https://github.com/Automattic/jetpack-genesis/issues/456

## Proposed Changes

* This PR implements animation for the illustration background for the new hosting page when switching tabs.

## Why are these changes being made?

* This change is mainly for the new hosting page, includes animation for the background. 

## Testing Instructions

* Open the A4A live link
* Visit Marketplace - Hosting. Verify you can see the animation when switching between tabs:

https://github.com/user-attachments/assets/7659d405-b0fb-4fcc-9fb6-0b1ad8eb93b4

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/a8c-for-agencies/sections/marketplace/hosting-overview/hosting-v2/style.scss [hunk 1]
```diff
@@ -1,16 +1,84 @@
...
+		overflow: hidden;
+
+		&::before {
+			background: url(calypso/assets/images/a8c-for-agencies/hosting-illustration.svg) no-repeat center/contain;
+			content: "";
+			display: inline-block;
+			position: absolute;
+			right: -150px;
+			top: -100px;
+			transition: transform 0.3s ease-in-out;
```
Comment (madebynoam) [inline client/a8c-for-agencies/sections/marketplace/hosting-overview/hosting-v2/style.scss]: @yashwin can you change this to .6s duration?

**Review by madebynoam — APPROVED** [review]
Lgtm! Thanks, @yashwin!
I just left a small note to change the duration to .6s to make it feel slightly smoother.

**Comment by yashwin** [comment]
> Lgtm! Thanks, @yashwin! I just left a small note to change the duration to .6s to make it feel slightly smoother.

Thanks for the review, @madebynoam! I have updated it to 0.6sec now: https://github.com/Automattic/wp-calypso/pull/93102/commits/5fa2420dd03b928de808460a91a54c352acb0171

**Review by jkguidaven — APPROVED** [review]
This looks awesome. Nice work! 👍

**Pull request merged**